include/clap/factory/draft/plugin-invalidation.h

```diff
@@ -1,7 +1,12 @@
 #pragma once
 
-#include "private/std.h"
-#include "private/macros.h"
+#include "../../private/std.h"
+#include "../../private/macros.h"
+
+// Use it to retrieve const clap_plugin_invalidation_factory_t* from
+// clap_plugin_entry.get_factory()
+static const CLAP_CONSTEXPR char CLAP_PLUGIN_INVALIDATION_FACTORY_ID[] =
+   "clap.plugin-invalidation-factory/draft0";
```

Comment:
```suggestion
   "clap.plugin-invalidation-factory/draft/0";
```
use same naming convention as draft extensions